README.md

```diff
@@ -1,26 +1,49 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# namecheap-lego-operator
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+## Description
 
-Use links instead.
--->
+Namecheap LEGO operator implements the provider side of the `tls-certificates-interface`
+to provide signed certificates from an ACME servers, using LEGO
+(https://go-acme.github.io/lego).
+It extends the `AcmeClient` charm from `acme_client` library to support Namecheap DNS API.
```

Comment:
This sentence(`It extends [...]`) is more for developers, I don't think it's necessary in the README.md